### PR TITLE
added new commits graph and removed previous link

### PR DESCRIPTION
**Pull request by subhadeepbanerjee2003**

### CODE REVIEW

**Comment by Arindam200** [comment]
Thanks a lot for the Contribution!

**Review by Arindam200 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -76,10 +76,6 @@ In this Blog, I will guide you on how to install and manage different nod... </p
...
  
 </table>
-</div>
-</td>
-</tr>
- <a href="http://www.github.com/Arindam200"><img src="https://github-readme-activity-graph.cyclic.app/graph?username=Arindam200&theme=merko&bg_color=1c1917&color=ffffff&line=0891b2&point=ffffff&area_color=1c1917&area=true&hide_border=true&custom_title=GitHub%20Commits%20Graph" alt="GitHub Commits Graph" /></a>
-</details>
 
 
+[![Arindam's github activity graph](https://github-readme-activity-graph.vercel.app/graph?username=Arindam200)](https://github.com/ashutosh00710/github-readme-activity-graph)
```
Comment (Arindam200) [inline README.md:84]:
```suggestion
</div>
</td>
</tr>
[![Arindam's github activity graph](https://github-readme-activity-graph.vercel.app/graph?username=Arindam200)](https://github.com/ashutosh00710/github-readme-activity-graph)
</details>
```

Comment (Arindam200) [reply README.md:84]: Please Change this part as suggested.

**Comment by Arindam200** [comment]
LGTM! 

Thanks again! 

Keep contributing!

**Comment by subhadeepbanerjee2003** [comment]
Thanks for giving me a chance to contribute 😁

**Pull request merged**